developers/level3.md

```diff
@@ -1,5 +1,5 @@
 # Engineer L3
-> Level 3 is the most needed team member. It does not mean we will limit the growth to higher levels, but on this level, you actually contribute in main areas of productivity for a developer. 
+> Level 3 is the most needed team member. It does not mean we will limit the growth to higher levels, but on this level, you actually contribute in main areas of productivity as a developer. 
```

Comment:
this is questionable, I'd say the original sentence is more clear in intent

Reply:
Let's ask more people

Reply:
Sentence structure is a bit blurry as a whole, but in this specific instance the word "as" makes more sense